### DISCUSSION TITLE
DeepEdit model to use outside MONAI Label

### DISCUSSION THREAD

**Discussion by Suchi97** [Project-MONAI/MONAILabel#888] [General]
Hi Team,

In MONAI Label, the deepedit app is used for interactive segmentation. So if we are using 8 labels, then the input channels will be 9 (labels + image) and the output channels will be 8 (which is the number of labels).

After annotating the images using deepedit app, the trained model expects 9 in_channels and the out_channels will be 8. 
Is there a way to use this deepedit trained model outside the MONAI Label. [Like, using this model as pretrained model for transfer learning]

Thanks in advance
Any help is appreciated.

**Comment by diazandr3s** [comment]
Hi @Suchi97,

Good question!
Yes, you can use the DeepEdit model outside MONAI Label. Here is a Notebook tutorial we've created showing how you can use deepedit model in inference mode: https://github.com/Project-MONAI/tutorials/blob/main/deepedit/ignite/infoANDinference.ipynb

[This transform](https://github.com/Project-MONAI/MONAI/blob/dev/monai/apps/deepedit/transforms.py#L615-L680) creates the channels needed for inference based on the number of labels.

I hope this helps,

**Comment by Suchi97** [comment]
Hi @diazandr3s ,

Thanks for your reply. 
This really helps!!

Thanks again @diazandr3s
